src/mpid/ch4/src/ch4r_rma.h

```diff
@@ -577,6 +581,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDIG_do_get_accumulate(const void *origin_addr,
 
     MPIDIG_REQUEST(sreq, rank) = target_rank;
     MPIDIG_REQUEST(sreq, req->areq.data_sz) = data_sz;
+    MPIDIG_REQUEST(sreq, req->areq.result_data_sz) = result_data_sz;
```

Comment:
We already store the `data_sz` with `MPIDIG_type_recv_init`, I don't think we need a separate field.

Reply:
We do need a separate field. The origin data_sz and result data_sz could be different.

Reply:
I will cleanup the name a bit to make it cleaner.